### PR TITLE
Add redis cache proposal

### PR DESCRIPTION
**Pull request by syed**

### CODE REVIEW

**Comment by BillDett** [comment]
Sounds like simple, non-sharded HA Redis is the way to go (and avoids any big client changes).  Will be very interesting to see what effect the network has on this over the intra-Pod memcache.  We might observe an overall increase in query latency while database traffic itself goes down.

File: enhancements/redis-cache.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+---
+
+# redis-cache-provider 
+
+Quay has a data caching mechanism which it uses to cache various objects.
+This proposal is to add Redis as another provider/implementation for this cache.
+
+`-`.
+
+To get started with this template:
```
Comment (alecmerdler) [inline enhancements/redis-cache.md:22]: Can you remove this part of the template?

Comment (alecmerdler) [inline enhancements/redis-cache.md:22]: Can you remove this part of the template?

**Review by alecmerdler — CHANGES_REQUESTED** [review]
Proposal looks good, just remove the sections of text left over from the template.

**Pull request merged**